pkg/local_object_storage/engine/evacuate.go

```diff
@@ -32,6 +35,11 @@ func (p *EvacuateShardPrm) WithIgnoreErrors(ignore bool) {
 	p.ignoreErrors = ignore
 }
 
+// WithHandler sets handler to call for objects which cannot be saved on other shards.
+func (p *EvacuateShardPrm) WithHandler(f func(oid.Address, *objectSDK.Object) error) {
```

Comment:
maybe expand naming with "Error" somehow? not clear what exactly that handler should handle

Reply:
It doesn't handle errors, it is called for objects which can't be put to other shards, which is what comment says.

Reply:
> It doesn't handle errors

yeah, i got it, did not think much about naming just wanted to highlight that `Handler` could be misunderstood since it is not a general handler: it is not called on every object, only called on "error" object. i would name that "FaultHandler" or smth like that

let it be up to you

Reply:
Agree, fixed.